server/src/main/java/org/candlepin/auth/KeycloakOIDCFacade.java

```diff
@@ -131,19 +129,21 @@ public String getRemoteAddr() {
 
         @Override
         public void setError(AuthenticationError error) {
+            // intentionally left empty
         }
 
         @Override
         public void setError(LogoutError error) {
-
+            // intentionally left empty
         }
     }
+
     /**
      * ResponseFacade for Keycloak Responses
      */
     protected class ResponseFacade implements OIDCHttpFacade.Response {
 
-        private javax.ws.rs.core.Response.ResponseBuilder responseBuilder =
+        private ResponseBuilder responseBuilder =
             javax.ws.rs.core.Response.status(204);
```

Comment:
Still fully qualifying the Response class here. Also, this all fits onto one line now.

Reply:
We cant shorten the Response class here as we need javax.ws.rs.core.Response.status because it conflicts with OIDCHttpFacade.Response